### ISSUE TITLE
eth_getBlockByNumber with true param returns invalid request

### ISSUE THREAD

**Issue by gagarin55** [ETCDEVTeam/sputnikvm-dev#21]
I've taken this request from MetaMask:
```
{id: 4645179836049334, jsonrpc: "2.0", params: ["0x52", true], method: "eth_getBlockByNumber"}
```

It returns 
```
{"jsonrpc":"2.0","error":{"code":-32600,"message":"Invalid request"},"id":4645179836049334}
```

**Comment by sorpaas** [comment]
That's most likely due to block 0x52 haven't been mined. Can you provide some context for this?

**Comment by gagarin55** [comment]
Yes, it might be a reason, but in case block doesn't exist it must return
```
{"jsonrpc":"2.0","result":null,"id":4645179836053265}
```

**Comment by sorpaas** [comment]
Okay, got it. I'll delay the NotFound/null issue for a while and fix it together with #2.
